application/helpers/expressions/em_manager_helper.php

```diff
@@ -21,6 +21,8 @@
     * @author Denis Chenu <http://sondages.pro>
     */
     Yii::import('application.helpers.expressions.em_core_helper', true);
+    Yii::import('application.helpers.expressions.em_cache_exception', true);
+    Yii::import('application.helpers.expressions.em_cache_helper', true);
```

Comment:
I like to see a good way for autoloader to work …

This don't work https://github.com/LimeSurvey/LimeSurvey/blob/2528b1178cc26c384ee519f56cf95cfb7ca4bf5a/application/helpers/questionHelper.php#L14 ?
viewHelper can not be autoloaded because it don't have the namespace.

Reply:
Arg … helpers inside subdirectory … maybe `namespace LimeSurvey\Helpers\expressions;`

Reply:
Separate issue, won't deal with in this PR.

Reply:
I can quikcly check : starting with good name space seems better in my opinion.

Reply:
A PR should be as small as possible. Not sure I agree with making changes to namespaces at the same time.

Reply:
Maybe when moving to develop then ?